### PR TITLE
[ISSUE #3122]Refactor JsonUtils

### LINKED ISSUE #3122: [Enhancement] Refactor JsonUtils
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

 Refactor JsonUtils

### Describe the solution you'd like

1. Consolidate the JsonUtils utility class in the project by merging the JsonUtils utility classes in the runtime module and the Common module.
2. Remove the JsonUtils utility class in the runtime module.
3. Add test cases for JsonUtils.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by mxsm**
Fixes #3122 

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

1. Consolidate the JsonUtils utility class in the project by merging the JsonUtils utility classes in the runtime module and the Common module.
2. Remove the JsonUtils utility class in the runtime module.
3. Add test cases for JsonUtils.

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java [hunk 1]
```diff
@@ -62,72 +63,72 @@ public static String serialize(Object obj) {
         }
     }
 
-    public static <T> byte[] serialize(String topic, Class<T> data) throws JsonProcessingException {
-        if (Objects.isNull(data)) {
+    public static byte[] toBytes(Object obj) {
```
Comment (jonyangx) [inline eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java:66]: Suggest use toJsonBytes is better.

File: eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java [hunk 2]
```diff
@@ -62,72 +63,72 @@ public static String serialize(Object obj) {
...
      *
-     * @param str           json string
+     * @param text          json string
      * @param typeReference object type reference
      * @param <T>           object type
      * @return object
      */
-    public static <T> T deserialize(String str, TypeReference<T> typeReference) {
-        if (StringUtils.isEmpty(str)) {
+    public static <T> T parseObject(String text, TypeReference<T> typeReference) {
```
Comment (jonyangx) [inline eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java:115]: Method name is inaccurate.Jackson api is readValue().

File: eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java (hunk 1 above)
Comment (mxsm) [reply eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java:66]: @jonyangx  I will fix it and resubmit later

**Comment by jonyangx** [comment]
@mxsm pls fix build error.

**Comment by mxsm** [comment]
@jonyangx  Done,  PTAL~ thks

File: eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java [hunk 3]
```diff
@@ -62,72 +63,72 @@ public static String serialize(Object obj) {
...
-     * @param clz  object class
-     * @param <T>  object type
+     * @param text  json string
+     * @param clazz object class
+     * @param <T>   object type
      * @return object
      */
-    public static <T> T deserialize(String json, Class<T> clz) {
-        if (StringUtils.isEmpty(json)) {
+    public static <T> T parseObject(String text, Class<T> clazz) {
```
Comment (jonyangx) [inline eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java:85]: @mxsm I think toObject() is better.

File: eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java [hunk 4]
```diff
@@ -39,10 +41,9 @@ public class JsonUtils {
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
     static {
-        OBJECT_MAPPER.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false)
-            .configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
         OBJECT_MAPPER.setSerializationInclusion(JsonInclude.Include.NON_NULL);
-
+        OBJECT_MAPPER.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+        OBJECT_MAPPER.disable(SerializationFeature.FAIL_ON_EMPTY_BEANS);
```
Comment (jonyangx) [inline eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java:46]: In eventmesh project should define default date format pattern in jsonutil. @mxsm 
eg. objectMapper.setDateFormat(new SimpleDateFormat("yyyy-MM-dd HH:mm:ss"))

Comment (mxsm) [reply eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java:46]: get it

**Review by jonyangx — APPROVED** [review]

**Pull request merged**